plugins/coral-plugin-offtopic/client/components/OffTopicFilter.js

```diff
@@ -1,31 +1,61 @@
 import React from 'react';
 import styles from './styles.css';
+import {t} from 'plugin-api/beta/client/services';
+import {PLUGIN_NAME, DEFAULT_CONFIG} from '../constants';
 
-export default class OffTopicFilter extends React.Component {
+/* plugin_config:
+  "coral-plugin-offtopic": {
+      enabled: true|false,
+      action_style: "checkbox|link",
+      use_tooltip: true|false
+    }
+  }
+
+*/
 
+export default class OffTopicFilter extends React.Component {
+  config = this.props.config;
+  pluginConfig = {...DEFAULT_CONFIG, ...(this.config && this.config[`${PLUGIN_NAME}`])};
```

Comment:
The 2 lines above should be: 

```js
getPluginConfig({config} = this.props) {
  return {...DEFAULT_CONFIG, ...(config && config[`${PLUGIN_NAME}`])};
}
```

Reasoning: The config is coming from the props, which can change over time. Turning this into the function above will always calculate the config from the most current props.